src/lib/rest/rest.cpp

```diff
@@ -1239,6 +1245,23 @@ static int connectionTreat
       return MHD_NO;
     }
 
+
+    // Get API version
+    ciP->apiVersion = (url[2] == '2')? V2 : V1;  // If an APIv2 request, the URL starts with "/v2/". Only V2 requests.
+
+    // LM_TMP(("--------------------- Serving APIv%d request %s %s -----------------", ciP->apiVersion, method, url));
```

Comment:
Leftover that should be removed?

Reply:
yup

Reply:
Or 'nope' would be a better answer.
The old line didn't contain API%d, that's all that has changed. Should be visible in the original diff ...
So, the line will stay, commented as always.

NTC